app/_locales/es/messages.json

```diff
@@ -3886,9 +3886,6 @@
     "message": "Seleccione esta opción para usar Etherscan para mostrar las transacciones entrantes en la lista de transacciones",
     "description": "$1 is the link to etherscan url and $2 is the link to the privacy policy of consensys APIs"
   },
-  "showIncomingTransactionsInformation": {
```

Comment:
would it be less regression work if we just change the name of this key like the one in `en`? because if the translation work didn't make the same release as this one, there will come some gap when rendering UI